kiwi/bootloader/config/grub2.py

```diff
@@ -827,20 +823,16 @@ def _setup_bios_image(self, mbrid=None, lookup_path=None):
             ['bash', '-c', bash_command]
         )
 
-    def _create_embedded_fat_efi_image(self):
-        Path.create(self.boot_dir + '/boot/' + self.arch)
-        efi_fat_image = ''.join(
-            [self.boot_dir + '/boot/', self.arch, '/efi']
-        )
+    def create_efi_loader_image(self, path):
         Command.run(
-            ['qemu-img', 'create', efi_fat_image, '20M']
+            ['qemu-img', 'create', path, '20M']
```

Comment:
I like the argument change and would make it even more explicit, e.g ```path_efi_fat_image``` or similar

Reply:
ok